File: 18.py
#!/usr/bin/python

from utils import *
import base64 as b64

cipher = 'L77na/nrFsKvynd6HzOoG7GHTLXsTVu9qvY/2syLXzhPweyyMTJULu/6/kXX0KSvoOLSFQ=='
cipher = b64.b64decode(cipher)

print aes_decrypt_ctr(cipher, 'YELLOW SUBMARINE', 0)

File: 18.go
package main

import (
	"./utils"
	"crypto/aes"
	"encoding/base64"
	"fmt"
)

func main() {
	nonce := 0
	key := []byte("YELLOW SUBMARINE")
	block, _ := aes.NewCipher(key)

	msg_enc := "L77na/nrFsKvynd6HzOoG7GHTLXsTVu9qvY/2syLXzhPweyyMTJULu/6/kXX0KSvoOLSFQ=="
	msg_dec, _ := base64.StdEncoding.DecodeString(msg_enc)

	res := utils.AesCtr(block, nonce, msg_dec)
	fmt.Println(string(res))
}
